Sources/SQL/Core/main.swift

```diff
@@ -0,0 +1,141 @@
+extension Table {
+    public static func f(field: Field) -> DeclaredField {
+        return self.field(field)
+    }
+}
+
+
+struct Artist {
+    let id: Int?
+    // Note the optionality. Fields that are marked `NOT NULL` in your schema should be non-optional.
```

Comment:
I'm not certain but my guess is that the original intent of the optionality was so one could instantiate a model and insert it without giving it a concrete primary key. If the id is _not_ optional, it means that you have to know the id before you can insert it, which in many cases can be impossible.